User_Service/.env

```diff
@@ -0,0 +1,8 @@
+host="0.0.0.0"
+user="furqan"
+password="furqan"
+db_name="User"
+port=5430
+JWT_SECRET="Furqan"
+Admin="Admin"
+Admin_pass="123"
```

Comment:
Generally env variable names are in all caps like HOST=, PASSWORD=

Reply:
will update the env file

Reply:
have made the necessary changes